packages/deploy-config-sub-generator/src/app/index.ts

```diff
@@ -69,6 +70,11 @@ export default class extends DeploymentGenerator implements DeployConfigGenerato
         this.target = parseTarget(args, opts);
         this.vscode = opts.vscode;
 
+        if (!this.env.isPackageRegistered(this.genNamespace)) {
```

Comment:
There's no real way to test this afaik, the yeoman test environment registers the namespace `gen:test` by default when running the generator. I can change the namespace but can't deregister it